README.md

```diff
@@ -1,208 +1,262 @@
 
-Click on the enabled checkbox and then click Save. Once saved click the restart button. The apple push notification system will start up and inform you that it has restarted. After you have done this you will be ready to send apple push notifications. Now that it is enabled the application will always start up when Mendix is started up.
+### Step 1 - Login to your hybrid mobile application
 
-<img src="assets/images/apn-step7.PNG"/>
+Open your hybrid mobile app and login to it.
 
+### Step 2 - Login as administrator to the web (desktop) application
 
-## Setting up Google Cloud Messaging Server
-In order to send google push notifications from this module you need to have set up a google account with google cloud messaging enabled.
-To do so follow these steps to get registered for Google cloud messaging and enter the details into the Mendix screens.
+Open your application in the browser and login as administrator.
 
-### Step 1 - Login to developers console
-Open up the google [developers console] (https://console.developers.google.com) and login with your google id.
-<img src="assets/images/gcm-step1.PNG"/>
+### Step 3 - Open the "Device Overview" page
 
-### Step 2 - Create project
-Click new project and fill in the project name and project ID for your application. Then click create. Once created you will see a project number at the top of the screen next to the project ID. Take note of this ID because you will need it later on for our sender ID. 
-<img src="assets/images/gcm-step2.PNG"/>
+Navigate to the `Device Overview` page. Here you should see that one device has already been registered, which is the device that you use to login to your hybrid mobile application. Continue by selecting the device and press the `New Message` button.
 
-### Step 3 - Enable an API
-Once created click on the button enable an API. 
-<img src="assets/images/gcm-step3.PNG"/>
+<img src="assets/images/testing/device overview.JPG"/>
 
-### Step 4 - Enable Google Cloud Messaging
-Locate the Google Cloud Messaging for Android and click the off button to turn on the API.
-<img src="assets/images/gcm-step4.PNG"/>
+### Step 4 - Send a message
 
-### Step 5 - Create server keys
-Click on the menu option credentials, located on the left hand side under the APIs & Auth section. Then click on the button under the public API Access that says create new key. When the popup appears press the Server key button and then press create on the next screen. Take note of the API Key because you will need this when we set up the push notifcation in Mendix.
-<img src="assets/images/gcm-step5.PNG"/>
+Fill in the title and the message on the form and press `Send immediate` button. You should see a new notification coming in your device. By default, if your hybrid mobile app is currently active, the message will be displayed in the app. Otherwise, it will be shown as a standard push notification.
```

Comment:
*in* the form (or page) IMO.

Suggestion: 'You device should receive a new notification.'

Not sure if we should use 'By default' here. This suggests it can be configured to behave differently, but we don't explain anywhere how. (And you can't, unless you mess with the widget, atm)